server/src/main/java/org/opensearch/search/internal/SearchContext.java

```diff
@@ -305,6 +305,10 @@ public final void assignRescoreDocIds(RescoreDocIds rescoreDocIds) {
 
     public abstract boolean trackScores();
 
+    public abstract SearchContext includeNamedQueriesScore(boolean includeNamedQueriesScore);
```

Comment:
This is a public API, we would need to have a `default` methods here

Reply:
sorry wdym by default methods here?

Reply:
i will just return "this" i guess to avoid breaking the API

Reply:
> i will just return "this" i guess to avoid breaking the API

Yeah, empty implementation is fine (meant that by `default` method, sorry for confusion), thank you